common/hashing_test.cpp

```diff
@@ -332,11 +332,15 @@ template <typename T>
 auto PrintFullWidthHex(llvm::raw_ostream& os, T value) {
   static_assert(sizeof(T) == 1 || sizeof(T) == 2 || sizeof(T) == 4 ||
                 sizeof(T) == 8);
+  // Given the nature of a format string and the good formatting, a nested
+  // conditional seems like the most readable structure.
+  // NOLINTBEGIN(readability-avoid-nested-conditional-operator)
```

Comment:
Maybe not good for this change, but had you considered:

```
os << llvm::formatv(llvm::formatv("{{0:x{0}}", 2 * sizeof(T)), static_cast<uint64_t>(value);
```

Reply:
Mostly it seems really unfortunate to have a dynamically computed format string rather than a string literal. But it's hard to get string literals without something like the nested conditions here.

Maybe it's just me, but I find the way `clang-format` renders the conditions to result in something very readable.

Reply:
FWIW, I was mostly thinking that since this is essentially debug code, the approach might just offer less to understand.